package.json

```diff
@@ -5,8 +5,8 @@
   "description": "End to end tests for a Form Builder form",
   "scripts": {
     "lint": "eslint .",
-    "test": "ava --reset-cache && ava -v",
-    "lint:test": "npm run lint && ava --reset-cache && ava -v",
+    "test": "ava --reset-cache && ava -v -s",
```

Comment:
This `-s` flag enforces `serial` for all specs, which means they all run in order and not in parallel. This slows the test suite but has significantly reduced random failures.